app/auth/routes.py

```diff
@@ -59,13 +59,74 @@ def register():
             "msg": "Error in request method"
         }), 400
 
+@app.route('/login/', methods=['POST'])
+def login():
+    if current_user.is_authenticated:
+        return jsonify({
+                    'error': False,
+                    'code': 200,
+                    'title': 'User is authenticated',
+                    'msg': 'User is authenticated'
+                }),200
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify({
+            'error': True,
+            'code': 400,
+            'title': 'Request failed',
+            'msg': str(e)
+            }), 400
+    username = data.get("username", None)
+    if not username:
+        return jsonify({
+            "error": True,
+            "code": 400,
+            "title": "Request failed",
+            "msg": "Username not found"
+            }), 400
+    user = User.find_by_name(username)
+    if not user:
+        return jsonify({
+            "error": True,
+            "code": 400,
+            "title": "Request failed",
+            "msg": "User not found"
+            }), 400
+    if not check_password_hash(user.password_hash, data['password']):
+        return jsonify({
+            'error': True,
+            'code': 400,
+            'title': 'Invalid password',
+            'msg': 'Password was wrong'
+            }), 400
+    else:
+        login_user(user)
+        return jsonify({
+            'error': False,
+            'code': 200,
+            'title': 'Login in succesful',
+            'msg': 'User is logged in'
+            }), 200
```

Comment:
again. The brackets should close in the same line with the beginning of the command